### PR TITLE
Online ifdef fixes

### PR DESCRIPTION
**Pull request by ahardin-rh**
@vikram-redhat Most of the fixes for the Architecture and Developer Guide build failures that we are seeing should be resolved with these fixes. For the others, those issues do not persist in the master branch, and so I will create a new PR against Online to fix those. 

@adellape PTAL

### CODE REVIEW

File: dev_guide/managing_images.adoc [hunk 1]
```diff
@@ -110,8 +110,14 @@ Metadata] for more details.
 
 If you want to instruct Docker to always fetch the tagged image from the
 integrated registry, use `--reference-policy=local`. The registry uses the
-xref:../install_config/registry/extended_registry_configuration.adoc#middleware-repository-pullthrough[pull-through
-feature] to serve the image to the client. By default, the image blobs are
+
```
Comment (adellape) [inline dev_guide/managing_images.adoc]: Extra blank line?

**Review by adellape — COMMENTED** [review]
LGTM

File: dev_guide/managing_images.adoc (hunk 1 above)
Comment (ahardin-rh) [inline dev_guide/managing_images.adoc]: woops! nice catch

**Comment by ahardin-rh** [comment]
Removing the Online label since https://github.com/openshift/openshift-docs/pull/4354 cherry-picks to that branch

**Comment by vikram-redhat** [comment]
Thanks @ahardin-rh

**Pull request merged**